### ISSUE TITLE
Automated testing error

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/griddle#18]
```
Error: Assertion failed: Capacity of data series exceeded
    at window.assertions.assertFunction (http://192.168.0.2:8080/assert/js/assert.js:22:13)
    at XYDataSeries.inherit.addPoint (http://192.168.0.2:8080/griddle/js/XYDataSeries.js?bust=1444431198917:61:17)
    at ScreenView.screenView.step (http://192.168.0.2:8080/griddle/js/griddle-main.js?bust=1444431198917:47:20)
    at animationLoop (http://192.168.0.2:8080/joist/js/Sim.js?bust=1444431198917:712:27)
```

**Comment by pixelzoom** [comment]
Still a problem as of a79a6f6069404d214ce4fc2b5700bafa82a9fb26.

**Comment by pixelzoom** [comment]
This occurs in requirejs mode for the griddle demo app, and occurs after the demo app has run for a minute or so.  Looks like there is a limit on number of data points that the demo isn't handling.

**Comment by jbphet** [comment]
This was due to an intended optimization made while working on the Neuron sim where the max size of the data series had to be specified.  Upon reflection, this probably isn't the best approach, and it would be better to have a default size and allow clients to specify a size via options, but just allocate more if the default or specified size is exceeded (the way that JavaScript does for arrays).  I've made that mod, removed the assertion, and the automated test is now succeeding.  Closing.
